posix-configs/SITL/init/rcS_gazebo_iris

```diff
@@ -71,5 +71,5 @@ mavlink stream -r 50 -s SERVO_OUTPUT_RAW_0 -u 14556
 mavlink stream -r 20 -s RC_CHANNELS -u 14556
 mavlink stream -r 250 -s HIGHRES_IMU -u 14556
 mavlink stream -r 10 -s OPTICAL_FLOW_RAD -u 14556
-mavlink boot_complete
 sdlog2 start -r 100 -e -t -a
+mavlink boot_complete
```

Comment:
Because that has been a mistake in the first place. The boot complete flag *has to* be the very last call in the startup script - always.